src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
```

Comment:
```suggestion
export function Person({
  person: {
    name,
    age,
    sex,
    isMarried,
    partnerName,  
  }
}) {
```
Destructurization on steroids!)